tasks/db.py

```diff
@@ -26,3 +26,10 @@ def get_metadata(engine):
     metadata = sqlalchemy.MetaData()
     metadata.reflect(bind=engine)
     return metadata
+
+
+def write_records(records, engine, table):
+    table.drop(engine, checkfirst=True)  # Overwrite table if it exists
+    table.create(engine)
+    with engine.begin() as conn:
+        conn.execute(sqlalchemy.insert(table), [record._asdict() for record in records])
```

Comment:
This is a matter of taste, but I don't think I would have added this function to this module: I think I would have added it to the `tasks.tasks.get_opencodelists_logins` module, where it's called (or where it will be called, eventually).

Would I be correct in thinking that you added it here because you thought we'd use it somewhere else in the future? If so, then please don't think too far into the future 🙂 That said, if you are thinking too far into the future, then future us would benefit from knowing that this function is destructive: it doesn't just write records; it also drops and creates the table.